generated-dockerfiles/rapidsai_centos7-devel.amd64.Dockerfile

```diff
@@ -15,49 +15,36 @@ ARG FROM_IMAGE=rapidsai/rapidsai-core-dev
 
 FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-devel-${LINUX_VER}-py${PYTHON_VER}
 
-ARG RAPIDS_VER
-ARG CUDA_VER
-ARG BUILD_BRANCH="branch-${RAPIDS_VER}"
+ARG DASK_SQL_VER
 
-ENV BLAZING_DIR=/blazing
+ENV DASK_SQL_DIR=/dask-sql
 
-RUN mkdir -p ${BLAZING_DIR} \
-    && cd ${BLAZING_DIR} \
-    && git clone https://github.com/BlazingDB/Welcome_to_BlazingSQL_Notebooks.git
 
-RUN gpuci_mamba_retry install -y -n rapids -c blazingsql-nightly -c blazingsql \
-      "blazingsql-build-env=${RAPIDS_VER}*" \
-      "rapids-build-env=${RAPIDS_VER}*" \
-      "cudatoolkit=${CUDA_VER}*" \
-    && gpuci_conda_retry remove -y -n rapids --force-remove \
-      "blazingsql-build-env=${RAPIDS_VER}*" \
-      "rapids-build-env=${RAPIDS_VER}*"
+RUN gpuci_mamba_retry install -y -n rapids -c conda-forge \
+      "maven>=3.6.0" \
+      "pip" \
+      "setuptools_scm"
 
-
-ENV CUDF_HOME=/rapids/cudf
-
-RUN mkdir -p ${BLAZING_DIR} \
-    && cd ${BLAZING_DIR} \
-    && git clone -b ${BUILD_BRANCH} https://github.com/rapidsai/blazingsql-release-staging.git blazingsql
+RUN mkdir -p ${DASK_SQL_DIR} \
+    && cd ${DASK_SQL_DIR} \
+    && git clone -b ${DASK_SQL_VER} https://github.com/dask-contrib/dask-sql dask-sql
 
 
 ENV LD_LIBRARY_PATH_ORIG=${LD_LIBRARY_PATH}
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/cuda/compat
 
-RUN rm -f ${GCC9_DIR}/lib64/libm.so.6
-
 RUN source activate rapids \
-    && cd ${BLAZING_DIR}/blazingsql \
-    && ./build.sh
+    && cd ${DASK_SQL_DIR}/dask-sql \
+    && python -m pip install . --no-deps -vv
```

Comment:
Do we also need to run `python setup.py java`, or is the Java installation covered here with the `pip install`?

Reply:
Tbh I don't know, I checked how to build `dask-sql` here: https://github.com/dask-contrib/dask-sql/blob/5b8f8a927fbd8416e52c154f6b4f680b0553fd55/continuous_integration/recipe/meta.yaml#L17
Please let me know if we need to do something else

Reply:
Ah yeah forgot about that - yeah that seems to trigger the Java build for the nightlies so we should be good here 😄